core/src/main/java/org/elasticsearch/cluster/routing/allocation/decider/AwarenessAllocationDecider.java

```diff
@@ -19,21 +19,20 @@
 
 package org.elasticsearch.cluster.routing.allocation.decider;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
My IDE is consistent, it just only moves when I optimize imports (usually to auto remove unused imports).

>  think we decided originally to be consistent with putting them below the ES imports?

Is this documented somewhere? I have intellij set to use the same order as eclipse defaults.